openedx/core/djangoapps/appsembler/sites/management/commands/offboard.py

```diff
@@ -425,7 +425,7 @@ def process_organization_users(self, organization):
         return [{
             'username': user.username,
             'active': user.is_active,
-        } for user in get_users_of_organization(organization=organization).all()]
+        } for user in get_users_of_organization(organization=organization, without_inactive_users=False).all()]
```

Comment:
`all()` is useless here.

```suggestion
        } for user in get_users_of_organization(organization=organization, without_inactive_users=False)]
```